### ISSUE TITLE
Begiak should tell #apertium about commits

### ISSUE THREAD

**Issue by ftyers** [apertium/phenny#386]
Begiak should publish commits (actually probably better that it just publishes "pushes") in apertium repos to #apertium on freenode.

**Comment by sushain97** [comment]
https://github.com/apertium/phenny/pull/385

**Comment by jonorthwash** [comment]
When I posted a comment to an issue, begiak's log did this:
```
----------------------------------------
Exception happened during processing of request from ('192.30.252.39', 59695)
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 313, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 341, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
TypeError: 'MyHandler' object is not callable
----------------------------------------
```

**Comment by jonorthwash** [comment]
And when I posted this just now it did this:
```
----------------------------------------
Exception happened during processing of request from ('192.30.252.39', 57483)
Traceback (most recent call last):
  File "/usr/lib/python3.5/socketserver.py", line 313, in _handle_request_noblock
    self.process_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 341, in process_request
    self.finish_request(request, client_address)
  File "/usr/lib/python3.5/socketserver.py", line 354, in finish_request
    self.RequestHandlerClass(request, client_address, self)
TypeError: 'MyHandler' object is not callable
----------------------------------------
```

**Comment by jonorthwash** [comment]
@sushain97, #385 is unrelated.

**Comment by jonorthwash** [comment]
#329 is the relevant issue.

**Comment by jonorthwash** [comment]
And probably #383.

**Comment by Androbin** [comment]
@jonorthwash Actually, I think it is related.
The `server_start` function got the wrong argument.
Should be fixed as of 9a843dacf1b29848db57ef036d79d7045bc28002.

**Comment by Androbin** [comment]
The `MyHandler` issue should be fixed. (Thanks @sushain97)
